portfolio.py

```diff
@@ -2,21 +2,47 @@
 Allocation of the asset.
 """
 
+import numpy as np
+from thousandaire.constants import TRADING_INSTRUMENTS
+
 class Portfolio(dict):
     """
     Set the portfolio
     """
-    def __init__(self, date):
+    def __init__(self, np_array=None, target=None):
```

Comment:
`target` is its context in alpha settings, but such context is not meaningful here.
The meaning of the argument here, is "how to decode the np_array?", which shouldn't be something named "target" in any sense.
I suggest to rename it to "encoding", which specifies the encoding method.

Reply:
Is this ignored again...?

Reply:
Oh! I have not push any renew version in this PR after last comments.